commands/usage.cmd

```diff
@@ -40,7 +40,8 @@ else
   WARDEN_ENV_PATH="$(locateEnvPath)" || true
   source "${WARDEN_DIR}/commands/usage.help"
   sourceCustomCommands "Your PC" "${WARDEN_HOME_DIR}/commands"
-  sourceCustomCommands "Environment \033[35m${WARDEN_ENV_PATH##*/}" "${WARDEN_ENV_PATH}/.warden/commands"
+  sourceCustomCommands "Environment (deprecated) \033[35m${WARDEN_ENV_PATH##*/}" "${WARDEN_ENV_PATH}/.warden/commands"
```

Comment:
```suggestion
  sourceCustomCommands "Environment (Warden) \033[35m${WARDEN_ENV_PATH##*/}" "${WARDEN_ENV_PATH}/.warden/commands"
```

It's a good idea, but I'm of the opinion this will make more sense to other users.

What do you think?